src/providers/okta.js

```diff
@@ -26,20 +26,43 @@ const Okta = {
 
     spinner.start();
 
-    let nightmare = Nightmare({show: process.env.DEBUG});
+    let samlAssertion = undefined;
+    let nightmare = Nightmare({
+      show: process.env.DEBUG,
+      openDevTools: true,
+      typeInterval: 20,
```

Comment:
I tried it with 5ms and my hyper-sensitive brain can tell that it's faster... ;-)